src/components/HomePage/SearchResultsArea/Offer/OfferDescription.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+
+import { useEditor, EditorContent } from "@tiptap/react";
+import StarterKit from "@tiptap/starter-kit";
+
+const OfferDescription = ({ content }) => {
+
+    const editor = useEditor({
+        extensions: [
+            StarterKit,
```

Comment:
Add the Underline extension here as well, to parse underline text. Just install @tiptap/extension-underline and then `import Underline from "@tiptap/extension-underline";`